Helm/opensearch/values.yaml

```diff
@@ -163,7 +157,18 @@ persistence:
   enabled: true
   labels:
     # Add default labels for the volumeClaimTemplate of the StatefulSet
-    enabled: false
+    enabled: false      
```

Comment:
May be worth removing this? I'm not sure why this exists in the first place.

Reply:
Is this issue resolved as of now? @DandyDeveloper @belfhi?

Reply:
Seems like there is no change to this anymore.